src/renderer/containers/Header/UserMenu.tsx

```diff
@@ -79,6 +89,16 @@ export const UserMenu: React.FC<{
             Refresh activation status
           </MenuItem>
         )}
+        {playKey && (
+          <MenuItem
+            onClick={() => {
+              closeMenu();
+              setOpenNameChangePrompt(true);
+            }}
+          >
+            Edit Name
```

Comment:
```suggestion
            Change display name
```

I think it might be better if it matched the modal title.